StoneMasonKarel.java

```diff
@@ -12,6 +13,93 @@
 
 public class StoneMasonKarel extends SuperKarel {
 
-	// You fill in this part
+	public void run() {
```

Comment:
Great job breaking down your solution into smaller parts. Awesome job for having a working solution.